lib/ZumoOta/src/WebServerCustom.h

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+    DESCRIPTION
+*******************************************************************************/
+/**
+ * @brief  WebServer realization
+ * @author Decareme Pauline Ngangnou <ngandeca@yahoo.fr>
+ *
+ * 
+ *
+ * @{
+ */
+
+#ifndef WebServerCustom_H
+#define WebServerCustom_H
+
+/******************************************************************************
+ * Includes
+ *****************************************************************************/
+#include <Arduino.h>

```

Comment:
include in header only what is needed by it. All others need to go into cpp

Reply:
comment above still open

Reply:
I will perform it with the next commit.